core/src/main/resources/db/migration/V2_0__init.sql

```diff
@@ -116,7 +116,7 @@ ALTER TABLE PUBLIC."deployments" ADD CONSTRAINT PUBLIC.CONSTRAINT_F PRIMARY KEY(
 CREATE INDEX PUBLIC."deployments_guid_idx" ON PUBLIC."deployments"("guid");    
 CREATE CACHED TABLE PUBLIC."audit_log"(
     "id" BIGINT DEFAULT (NEXT VALUE FOR PUBLIC.SYSTEM_SEQUENCE_15FD08D0_DDC1_4ECB_8FB3_38AB48CE23B4) NOT NULL NULL_TO_DEFAULT SEQUENCE PUBLIC.SYSTEM_SEQUENCE_15FD08D0_DDC1_4ECB_8FB3_38AB48CE23B4,
-    "release_id" BIGINT,
+    "release_id" VARCHAR(25),
```

Comment:
AFAICS this is the only change affecting de data model so far as `release_id` was already defined as `VARCHAR(25)` in the `releases` table

Reply:
@juanjovazquez we should never be changing an existing migrations. If you want to change the DB scheme you should create  a new migration. Flyaway uses a numbering convention in the name, so you'd create a filed called V2_1__alter_relase_id.sql where you alter the "audit_log" table.

Reply:
@kaiserpelagic Got it. I'll change it right away.

Reply:
@juanjovazquez it seems like we also need a migration for the `releases` table.

Reply:
@kaiserpelagic Why is that?. `release_id` was already a `VARCHAR(25)` in the `releases` table.

Reply:
@juanjovazquez interesting, I did not realize that

Reply:
@kaiserpelagic yep, I got surprise too ;-)